pkg/v22/configmap/create.go

```diff
@@ -44,7 +44,7 @@ func (s *Service) newCreateChange(ctx context.Context, currentConfigMaps, desire
 
 	for _, desiredConfigMap := range desiredConfigMaps {
 		appName := desiredConfigMap.Labels[label.App]
-		chartSpec := s.getChartSpecByName(appName)
+		chartSpec := s.getChartSpecByAppName(appName)
```

Comment:
We get the chart spec to see if it has been migrated. If so we don't create the old style configmaps.